ide/docker.api/src/org/netbeans/modules/docker/api/DockerSupport.java

```diff
@@ -129,12 +128,11 @@ public void removeChangeListener(ChangeListener listener) {
     }
 
     public boolean isSocketSupported() {
-        return false;
-//        if (BaseUtilities.getOperatingSystem() != BaseUtilities.OS_LINUX) {
-//            return false;
-//        }
-//        String arch = System.getProperty("os.arch"); // NOI18N
-//        return arch != null && (arch.contains("x86") || arch.contains("amd64")); // NOI18N
+        if (BaseUtilities.getOperatingSystem() != BaseUtilities.OS_LINUX) {
```

Comment:
What about MacOS X ? `junixsocket`'s documenation says it supports mac too...

Reply:
I stated this at the very top already: I don't have ability to verify this on `linux/aarch64` and `macos`

Reply:
Is there a reason to doubt, that this will work on Mac OS X? If not I would enable it there also (I'm assuming, that docker on Mac OS X also uses unix domain sockets).

Reply:
addressed (afaik MacOs does allow connection via unix domain sockets)